documentation/Get-Bootchart-metrics.md

```diff
@@ -0,0 +1,91 @@
+# Get Bootchart from a Clear Container #
+
+The following steps describe how to get a bootchart from a Clear Containers
+container. Currently `systemd` is the default init process in the Clear Containers mini-os
+system. The Systemd project provides a tool called `systemd-bootchart` that
+collects the CPU load, disk load, memory usage, as well as per-process information
+from the os boot process.
```

Comment:
Is the per-process information coming **from** the the OS boot process or is the information **about** the OS boot process? Also keep an eye out for capitalization: OS is the acceptable way to abbreviate operating system.